R/theme_gtsummary.R

```diff
@@ -122,3 +122,16 @@ theme_gtsummary_printer <- function(
 
   list("pkgwide-str:print_engine" = match.arg(print_engine))
 }
+
+# ------------------------------------------------------------------------------
+#' @rdname theme_gtsummary
+#' @param language String indicating language. Must be one of `"es"` (Spanish)
+#' or `"en"` (English).
+#' @export
+theme_gtsummary_language <- function(language = c("es", "en")) {
```

Comment:
we discussed that you could potentially remove `en`, but not necessary.

Reply:
I think that the `en` option should be keeped, is the only way (at least I did not find another) to keep the choice for the user to get back to english if the language has been changed

Reply:
Ahh, that is a good point!